lib/babe/verify_integration_test.go

```diff
@@ -481,13 +461,13 @@ func TestVerifyAuthorshipRight_Equivocation(t *testing.T) {
 	err = babeService.blockState.AddBlock(block)
 	require.NoError(t, err)
 
-	err = verifier.verifyAuthorshipRight(&block.Header)
+	err = verificationManager.VerifyBlock(&block.Header)
 	require.NoError(t, err)
 
 	err = babeService.blockState.AddBlock(block2)
 	require.NoError(t, err)
 
-	err = verifier.verifyAuthorshipRight(&block2.Header)
+	err = verificationManager.VerifyBlock(&block2.Header)
```

Comment:
This test is meant to test `verifyAuthorshipRight`, we can't call `VerifyBlock` directly. We must be calling `verifyAuthorshipRight`

Reply:
So with how we use the VerificationManager in babe, the verifies is a private object so cannot retrieve it to call this receiver which is why I call VerifyBlock. Should I export or create a getter in order to call this specific function? Open to ideas on how to do this